### PR TITLE
Modified script Sep 06

### PR DESCRIPTION
**Pull request by manjunath-grl**
issue_id:  [#450](https://github.com/CHIP-Specifications/chip-test-scripts/issues/450), [#460](https://github.com/CHIP-Specifications/chip-test-scripts/issues/460), [#453](https://github.com/CHIP-Specifications/chip-test-scripts/issues/453) and [#459](https://github.com/CHIP-Specifications/chip-test-scripts/issues/459) 
Modified Manual and automation script
Test_TC_CC_3_1
Test_TC_CC_3_2
Test_TC_CC_3_3
Test_TC_CC_4_1
Test_TC_CC_4_2
Test_TC_CC_4_3
Test_TC_CC_4_4
Test_TC_CC_4_5
Test_TC_CC_5_2
Test_TC_CC_5_4
Test_TC_CC_6_1
Test_TC_CC_7_5
Test_TC_DRLK_2_2
Test_TC_G_3_2
Test_TC_IDM_4_1
Test_TC_LTIME_1_1
Test_TC_LUNIT_1_1
Test_TC_LVL_2_3
Test_TC_LVL_8_1
Test_TC_OPCREDS_3_3
Test_TC_SC_4_7
Test_TC_ULABEL_3_1
Test_TC_OO_1_1
Test_TC_DGTHREAD_1_1
Test_TC_DGTHREAD_2_1

Modified manual and automation script Sep 6
Test_TC_BIND_2_3.yaml
Test_TC_CADMIN_1_1.yaml
Test_TC_CADMIN_1_2.yaml
Test_TC_CADMIN_1_7.yaml
Test_TC_CADMIN_1_8.yaml
Test_TC_CADMIN_1_9.yaml
Test_TC_CADMIN_1_10.yaml
Test_TC_CADMIN_1_11.yaml
Test_TC_CADMIN_1_12.yaml
Test_TC_CADMIN_1_13.yaml
Test_TC_CADMIN_1_14.yaml
Test_TC_CADMIN_1_15.yaml
Test_TC_CADMIN_1_16.yaml
Test_TC_CADMIN_1_17.yaml
Test_TC_CADMIN_1_18.yaml
Test_TC_CNET_1_3.yaml
Test_TC_CNET_4_1.yaml
Test_TC_CNET_4_2.yaml
Test_TC_CNET_4_3.yaml
Test_TC_CNET_4_10.yaml
Test_TC_CNET_4_11.yaml
Test_TC_CNET_4_12.yaml
Test_TC_CNET_4_15.yaml
Test_TC_CNET_4_16.yaml
Test_TC_CNET_4_17.yaml
Test_TC_CNET_4_18.yaml
Test_TC_CNET_4_19.yaml
Test_TC_CNET_4_20.yaml
Test_TC_CNET_4_21.yaml
Test_TC_DA_1_1.yaml
Test_TC_DA_1_4.yaml
Test_TC_DA_1_6.yaml
Test_TC_DA_1_7.yaml
Test_TC_DD_1_5.yaml
Test_TC_DD_1_8.yaml
Test_TC_DD_1_9.yaml
Test_TC_DD_1_13.yaml
Test_TC_DD_2_1.yaml
Test_TC_DD_3_1.yaml
Test_TC_DD_3_5.yaml
Test_TC_DD_3_8.yaml
Test_TC_DD_3_9.yaml
Test_TC_DD_3_17.yaml
Test_TC_DD_3_19.yaml
Test_TC_DGGEN_3_1.yaml
Test_TC_DGTHREAD_3_1.yaml
Test_TC_DGTHREAD_3_2.yaml
Test_TC_DGTHREAD_3_3.yaml
Test_TC_DGTHREAD_3_4.yaml
Test_TC_DGWIFI_3_1.yaml
Test_TC_DGWIFI_3_2.yaml
Test_TC_IDM_6_1.yaml
Test_TC_IDM_6_2.yaml
Test_TC_IDM_8_1.yaml
Test_TC_OCC_2_4.yaml
Test_TC_OPCREDS_3_1.yaml
Test_TC_OPCREDS_3_4.yaml
Test_TC_OPCREDS_3_5.yaml
Test_TC_PS_2_2.yaml
Test_TC_PSCFG_2_2.yaml
Test_TC_ULABEL_3_1.yaml
Test_TC_WNCV_5_1.yaml
Test_TC_WNCV_7_1.yaml
Added auto generated files

Tested
Attached execution logs:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/9497388/Execution_log.txt)

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ tests:
     - label:
           "Given DGTHREAD.S.F00(PKTCNT) ensure featuremap has the correct bit
           set"
-      PICS: DGETH.S.F00
+      PICS: DGTHREAD.S.F00
```
Comment (yinyihu-silabs) [inline src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:57]: `DGTHREAD.S.F00` does not exist in PICS v0.11.

File: src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml [hunk 2]
```diff
@@ -65,7 +65,7 @@ tests:
     - label:
           "Given DGTHREAD.S.F01(ERRCNT) ensure featuremap has the correct bit
           set"
-      PICS: DGETH.S.F01
+      PICS: DGTHREAD.S.F01
```
Comment (yinyihu-silabs) [inline src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:68]: `DGTHREAD.S.F01` does not exist in PICS v0.11.

**Review by yinyi-hu — CHANGES_REQUESTED** [review]

File: src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:57]: @yinyihu-silabs testplan tells to use PICS DGTHREAD.S.F00 
image

File: src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml (hunk 2 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:68]: @yinyihu-silabs Script is as per testplan to use PICS: DGTHREAD.S.F01 in step 3.
image

Comment (yinyihu-silabs) [reply src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:68]: @manjunath-grl 
PICS xml 0.11 does not have the same PICS above
image

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml [hunk 3]
```diff
@@ -42,8 +42,8 @@ tests:
 
     - label: "TH reads the FeatureMap from DUT"
       PICS:
-          " !DGTHREAD.S.F00 && !DGTHREAD.S.F00 && !DGTHREAD.S.F00 &&
-          !DGTHREAD.S.F00 "
+          " !DGTHREAD.S.F00 && !DGTHREAD.S.F01 && !DGTHREAD.S.F02 &&
```
Comment (yinyihu-silabs) [inline src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:45]: DGTHREAD.S.F00 and DGTHREAD.S.F01 do not exist in PICS v0.11 xml.
are we updating PICS xml or yaml?

Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:45]: @yinyihu-silabs these changes are as part of recent updates in the test plan  [PR](https://github.com/CHIP-Specifications/chip-test-plans/pull/2216) @raju-apple

File: src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:68]: @yinyihu-silabs these changes are as part of recent updates in the test plan  [PR](https://github.com/CHIP-Specifications/chip-test-plans/pull/2216) @raju-apple

[the remark this answers is not in the document]
Comment (kvsmohan) [reply src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:68]: @yinyihu-silabs Thread Network Diagnostics test plan updated.
https://github.com/CHIP-Specifications/chip-test-plans/pull/2216
I have generated the new XML file,
image
[Thread Network Diagnostics Cluster Test Plan.zip](https://github.com/project-chip/connectedhomeip/files/9537651/Thread.Network.Diagnostics.Cluster.Test.Plan.zip)

File: src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml (hunk 3 above)
Comment (kvsmohan) [reply src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml:45]: @yinyihu-silabs 
image

**Comment by kvsmohan** [comment]
Updated XML File:
[Thread Network Diagnostics Cluster Test Plan.zip](https://github.com/project-chip/connectedhomeip/files/9537661/Thread.Network.Diagnostics.Cluster.Test.Plan.zip)

**Review by yinyi-hu — APPROVED** [review]

**Pull request merged**